pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -234,3 +254,84 @@ format_annot <- function(annot_list) {
   rownames(annot) <- annot$input
   return(annot)
 }
+
+# Fix annotations makes annotations compatible between samples with different types.
+# The possible options for features_types at this stage are 0,1,2
+#  0 is SYMBOL/ SYMBOL
+#  1 is IDS/SYMBOL
+#  2 is IDS/IDS
+equalize_annotation_types <- function(annot_list, counts_list, features_types_list, samples) {
+  if (any(features_types_list == 2) && any(features_types_list == 0) && !any(features_types_list == 1)) stop("Incompatible features detected.")
+
```

Comment:
```suggestion
  if (any(features_types_list == 2) &&
      any(features_types_list == 0) &&
      !any(features_types_list == 1)) {
      stop("Incompatible features detected.")
}

```

Reply:
all the hardcoded values should be constants. e.g.:
```
if (any(features_types_list == IDS_IDS) &&
	      any(features_types_list == SYM_IDS) &&
	      !any(features_types_list == SYM_SYM)) {
```

Reply:
I like this idea. Let's add this to sysdata.R

Reply:
done